meta-mender-core/recipes-bsp/u-boot/u-boot-mender-common.inc

```diff
@@ -290,6 +290,7 @@ SRC_URI_append_mender-uboot = "${@bb.utils.contains('MENDER_UBOOT_AUTO_CONFIGURE
                                                     ' file://uboot_auto_configure.sh \
                                                       file://uboot_auto_patch.sh \
                                                       file://add_kconfig_option_with_depends.py', \
+                                                      file://boolean.py', \
```

Comment:
You've got a typo here: Only one of the lines should have the terminating `'`.

Reply:
Oh sorry for overlooking it -> b070331

Reply:
That was quick, thanks!